tools/releaseBuild/azureDevOps/AzArtifactFeed/PSGalleryToAzArtifacts.yml

```
@@ -8,7 +8,11 @@ queue:
   name: Hosted VS2017
 steps:
   - pwsh: |
-      Install-Module -Name PowerShellGet -MinimumVersion 2.0.1 -Force
+      $minVer = [version]::new("2.2.3")
```

Possible suggestion:

```suggestion
      $minVer = [version]"2.2.3"
```

---

This might even work without the version type, given that it's on the RHS of the comparison